smf/kbmd

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
+
+set -o xtrace
+
+. /lib/svc/share/smf_include.sh
+. /lib/sdc/config.sh
+
+PATH=/usr/sbin:/usr/bin; export PATH
+
+# We are running early enough in the boot process, that
+# the /dev entries for any ccid entries (including /dev/ccid
+# itself) don't show up.  Running devfsadm first seems
+# to solve this, though perhaps there's a better
+# method?
+devfsadm
```

Comment:
It appears to be so -- trying to get privilege dropping working (though I may end up having to abandon it if it doesn't yield results soon), I added a 'pivy-tool' list prior to running devfsadm in the method script, and it cannot see the YK until after devfsadm is run. I'm not sure what's going on, but also not sure if it's worth tracking down right now.

Reply:
Reading the comment I said to myself "there is a better way: a dependency on `milestone/devices`."  It seems you have that and it still isn't working.  To save the next person a bit of time, it would be best to mention that the dependency on `milestone/devices` seems to not be sufficient.

Reply:
confirmed